src/SLCore.UnitTests/Core/SLCoreServiceProviderTests.cs

```diff
@@ -22,7 +22,7 @@
 using SonarLint.VisualStudio.SLCore.Core;
 using SonarLint.VisualStudio.TestInfrastructure;
 
-namespace SonarLint.VisualStudio.SLCore.UnitTests;
+namespace SonarLint.VisualStudio.SLCore.UnitTests.Core;
```

Comment:
see comment on SLCoreListenerSetUpTests. (This also had some whitespace fixes done automatically)